src-cpp-structure/projects/App/Global/LocalAccountSettings.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "SettingsProperties.h"
+
+#include <QtCore>
+
+namespace Status
+{
+    namespace LocalAccountSettingsKeys {
+        const QString storeToKeychain = "storeToKeychain";
+        const QString isKeycardEnabled = "isKeycardEnabled";
+
+        QVariant getDefaultValue(const QString& key);
+    }
+
+    namespace LocalAccountSettingsPossibleValues {
+        namespace StoreToKeychain {
+            const QString Store = "store";
+            const QString NotNow = "notNow";
+            const QString Never = "never";
+        }
+    }
```

Comment:
What's a reason to add string constants to `h` files?

Reply:
actually they are part of the appropriate namespace and added to .h in order to be available on the place we want to use that settings